sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/DefaultWizard.java

```diff
@@ -779,7 +779,7 @@ private Map<String, String> getDistroVersionsOptionsMap(Set<String> versions, Ve
 	private Map<String, String> getO3VersionsOptionsMap(VersionsHelper versionsHelper,
 			String optionTemplate, String artifactTemplate) {
 		Map<String, String> optionsMap = new LinkedHashMap<>();
-		Artifact artifact = new Artifact("referenceapplication-distro", "3.0.0-SNAPSHOT", "org.openmrs.distro", "zip");
+		Artifact artifact = new Artifact("referenceapplication-distro", null, "org.openmrs.distro", "zip");
```

Comment:
I remember suggesting passing NULL for the version parameter and you said it was not allowed.

Reply:
Apologies for the confusion. When evaluating with the debugger I encountered an exception when using null. So I decided not to use null. Upon further investigation, I realized that even though an exception gets thrown, the code was still able to fetch the versions list.

Reply:
Can we avoid the exception being thrown?

Reply:
[@dkayiwa](https://github.com/dkayiwa) I am sorry, I have made a mistake. I accidentally ran
 mvn org.openmrs.maven.plugins:openmrs-sdk-maven-plugin:4.6.0-SNAPSHOT:setup instead of mvn org.openmrs.maven.plugins:openmrs-sdk-maven-plugin:4.7.0-SNAPSHOT:setup when testing new code changes.
We can't still use null. If we pass null we get the following error.
[ERROR] Failed to execute goal org.openmrs.maven.plugins:openmrs-sdk-maven-plugin:4.7.0-SNAPSHOT:setup (default-cli) on project openmrs-sdk: Failed to setup server: For artifact {org.openmrs.distro:referenceapplication-distro:null:}: The version cannot be empty.
I am changing the parameter to "version".

Reply:
@dkayiwa I am really sorry, I have made a mistake. I accidentally ran 
`mvn org.openmrs.maven.plugins:openmrs-sdk-maven-plugin:4.6.0-SNAPSHOT:setup` instead of `mvn org.openmrs.maven.plugins:openmrs-sdk-maven-plugin:4.7.0-SNAPSHOT:setup` when testing new code changes.

We can't still use `null`. If we pass null we get the following error.

`[ERROR] Failed to execute goal org.openmrs.maven.plugins:openmrs-sdk-maven-plugin:4.7.0-SNAPSHOT:setup (default-cli) on project openmrs-sdk: Failed to setup server: For artifact {org.openmrs.distro:referenceapplication-distro:null:}: The version cannot be empty.`

I am changing the parameter to `"version"`.